app/src/main/java/com/antosik/benchproject/app/common/binding/swipeRefresh/ViewBindingAdapters.kt

```diff
@@ -0,0 +1,9 @@
+package com.antosik.benchproject.app.common.binding.swipeRefresh
+
+import androidx.databinding.BindingAdapter
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+
+@BindingAdapter("isStartingRefresh")
+fun SwipeRefreshLayout.isRefreshing(flag: Boolean) {
```

Comment:
Avoid naming like `flag`, name it what it really is 😄

Reply:
Also please check if this BindingAdapter is really needed. Maybe you can just use `isRefreshing` in XML because it is a public field in `SwipeRefreshLayout`

Reply:
Ok, I understand and it will be no longer in the code because I deleted it :D